SharedPluginFiles/FormItPluginStyling.css

```diff
@@ -20,51 +20,29 @@ body {
     flex-grow: 1;
 }
 
-/* the footer: information about plugins, and this plugin specifically */
-.footerDiv {
-    background-color: hsl(0, 0%, 97%);
-    font-family: sans-serif;
-    padding: 15px;
-}
-
-.footerContainer {
-    flex-shrink: 0;
-    width: 100%;
-    font-size: 16px;
-    margin-top: 10px;
-}
-
-.footerContainer p {
-    margin-bottom: 0;
-}
-
-.footerContainer a {
-    color: #4676a9;
-    text-decoration: none;
-}
-
-.footerContainer a:hover {
-    color: rgb(158, 189, 199);
-}
-
-.footerContainer a:visited {
-    color: #4676a9;
+/* typical body text */
+.contentContainer p, 
+.contentContainer ul,
+#pluginFooterContainer ul {
+    font-family: Arial;
+    line-height: 20px;
 }
 
 /* unordered lists */
-ul {
+.contentContainer ul,
+#pluginFooterContainer ul {
     margin-bottom: 0;
 }
 
-li {
+.contentContainer li {
```

Comment:
I know that .contentContainer should use an id instead of a class, but that's going to require changes to other plugins and I didn't want to include it with this PR